### DISCUSSION TITLE
Can't get foreground_process_name

### DISCUSSION THREAD

**Discussion by kjurl** [wezterm/wezterm#4707] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

I cannot get the `foreground_process_name`. I was following [this](https://github.com/wez/wezterm/issues/876#issuecomment-1000605626) and cannot get it to work. Through a bit of digging around it seems that `tab.active_pane.foreground_process_name` returns `""`. If I am running nvim in a tab, how do I get that process name (`nvim`).

I have source [this](https://github.com/wez/wezterm/blob/main/assets/shell-integration/wezterm.sh) shell script. Am using Bash. Wezterm is installed through flatpak.

**Comment by wez** [comment, marked as answer]
> Wezterm is installed through flatpak.

From https://wezfurlong.org/wezterm/install/linux.html#installing-on-linux-via-flathub

![image](https://github.com/wez/wezterm/assets/117777/1f904a1d-586c-41a0-9ef1-d59cbc30d654)

Process names from outside the flatpak sandbox are also not visible inside the sandbox

Resolution: install a native package on your system
